package.json

```diff
@@ -0,0 +1,196 @@
+{
+  "name": "vite-starter",
```

Comment:
I am not aware of this pattern of having a theme-specific package.json in the project root. Can you please point me to references for this?

Further, what about the dependencies defined currently in the theme directory? How are they used right now?